### ISSUE TITLE
Validate final uploads to mitigate basic problems

### ISSUE THREAD

**Issue by lordmauve** [pyweekorg/pyweekorg#27]
Create a downloadable "linter" to check the archive is correct before upload.

* discourage mixed-case filenames (tend to be Windows users unaware that this won't work on Linux)
* tarbombs
* archive types that are not .zip
* 'pyweek.*', \d+ from zipfile top level directories
* Require a requirements.txt
* Require a README.*
* Require a run_game.py

We can run the linter on the server and "disqualify" uploads that do not satisfy the criteria. (This might be too harsh, but can tune this).

**Comment by rdb** [comment]
I wouldn't disqualify a user for not following these rules, but it would be useful to have such a checking tool.  But I would also suggest adding this information to a hints section on the website or even to the rules page, since ideally users would already know about it *before* they make their archive.

**Comment by lordmauve** [comment]
Yeah, I agree it's too harsh. I'm jet-lagged and just wanted to get some thoughts down.

I think improving the consistency of the entries would make it easier for judges. Also I'd like to discourage entries that don't include source.
